### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by eggplants**
Fixes flake8 errors for passing CI.

### CODE REVIEW

**Comment by aucampia** [comment]
Type fixes are already made here: https://github.com/RDFLib/rdflib/pull/1759/files

Review welcome.

Please try and keep PRs limited in scope.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -14,21 +14,10 @@ repos:
       pass_filenames: false
       require_serial: true
       args: ["."]
-  - repo: local
+  - repo: https://github.com/psf/black
```
Comment (aucampia) [inline .pre-commit-config.yaml:17]: Could the black changes not go in a separate PR?

Comment (eggplants) [reply .pre-commit-config.yaml]: Created. #1763
